src/renderer/views/Playlist/Playlist.vue

```diff
@@ -107,7 +107,7 @@
               :initial-visible-state="index < 10"
               @move-video-up="moveVideoUp(item.videoId, item.playlistItemId)"
               @move-video-down="moveVideoDown(item.videoId, item.playlistItemId)"
-              @remove-from-playlist="removeVideoFromPlaylist(item.videoId, item.playlistItemId)"
+              @remove-from-playlist="removeVideoFromPlaylist(item.videoId)"
```

Comment:
Maybe you can pass item here so you dont need to find the video in the remove function?

Reply:
item.playlistItemId is giving null value